packages/legacy/core/App/localization/fr/index.ts

```diff
@@ -344,8 +358,13 @@ const translation = {
         "ConfirmDeclinedTitle": "Êtes-vous sûr de vouloir refuser ce justificatif?",
         "ConfirmDeclinedMessage": "Afin de recevoir à nouveau ce justificatif, vous devrez présenter une nouvelle demande auprès de l'émetteur.",
         "AbortDecline": "Non, revenir en arrière",
-        "DeclineTitle": "Refuser le justificatif",
+        "DeclineTitle": "Refuser l'offre de justificatif d'identité?",
         "Declined": "Diminué",
+        "DeclineParagraph1": "Afin de recevoir à nouveau l'offre de justificatif d'identité, vous devrez recommencer le processus d'émission avec le service d'émission.",
+        "DeclineParagraph2": "Voulez-vous vraiment refuser cette offre de justificatif d'identité?",
+        "CustomOfferTitle": "Supprimer cette offre?",
+        "CustomOfferParagraph1": "La suppression de cette offre supprimera la notification de votre liste.",
+        "CustomOfferParagraph2": "Vous ne reconnaissez pas l'organisation ? Vérifiez votre liste de contacts. vous ne recevez des notifications que des contacts que vous avez initiés."
```

Comment:
Grammar mistake, extra space between "l'organisation" and "?" lowercase "v" in "Vous" after the period.